package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "canvas",
   "description": "Canvas graphics API backed by Cairo",
-  "version": "2.10.1",
+  "version": "2.10.2",
```

Comment:
We increment this when we're making a release and create a tag at the same time. Can you undo this line please?